DolbyIO/Source/Private/DolbyIOSubsystem.cpp

```diff
@@ -2,28 +2,31 @@
 
 #include "DolbyIOSubsystem.h"
 
+#include "DolbyIOConversions.h"
 #include "DolbyIOLogging.h"
 #include "DolbyIOVideoSink.h"
 
 #include "Async/Async.h"
 #include "Engine/World.h"
 #include "GameFramework/PlayerController.h"
-#include "Misc/Base64.h"
 #include "TimerManager.h"
 
+using namespace dolbyio::comms;
+using namespace DolbyIO;
```

Comment:
Actually I don't like `using namespace` for big context (like file) and prefer `using Namespace::Type` as more descriptive. As cppsdk uses different naming convention using namespace for it is fine (and it shortens lines noticeably).
For `DolbyIO` I would suggest `using Namespace::Type` for most frequent types (like `DolbyIO::ToStdString` or `DolbyIO::FVideoSink`), but I will understand if you leave it as is. ;)